snapshots/tuberculosis/2023-11-27/burden_estimates.csv.dvc

```diff
@@ -3,11 +3,11 @@
 meta:
   origin:
     # Data product / Snapshot
-    title: Global tuberculosis report
+    title: Global Tuberculosis Report
     description: |-
       WHO has published a global tuberculosis (TB) report every year since 1997. The report provides a comprehensive and up-to-date assessment of the TB epidemic, and of progress in prevention, diagnosis and treatment of the disease at global, regional and country levels.
     date_published: 2023-11-07
-    title_snapshot: Global tuberculosis report - burden estimates
+    title_snapshot: Global Tuberculosis Report - Burden Estimates
```

Comment:
A couple of comments on the rest of this file:
* The license should not appear in citation_full.
* The license.url should be the place within the WHO website where they explain their license, not the explanation of the license itself.

Reply:
The same applies to the other dvc file.

Reply:
Ah okay - the source includes the licence in their citation, so I wasn't sure. I should still remove it?

Reply:
If they are very explicit about how to cite them, then follow their guidelines. Otherwise, it's better to remove the license from there (since it's defined elsewhere). Given that this citation is propagated, and the license of the data can change down the line, it's better to not mention any license outside the appropriate fields.